src/go/rpk/pkg/cli/cmd/start.go

```diff
@@ -80,7 +80,10 @@ func NewStartCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 	prestartCfg := prestartConfig{}
 	var (
 		configFile      string
+		nodeID          uint
```

Comment:
i see, so then you just rehydrate this from the docker api.? or you infer the node id from the `~/.rpk/cluster/node-<id>`